Robot_Adapter/Convert/ToRobot/Loads/Load.cs

```diff
@@ -341,7 +336,7 @@ public static void RobotLoad(this oM.Structure.Loads.ContourLoad load, RobotSimp
 
         /***************************************************/
 
-        public static void RobotLoad(this oM.Structure.Loads.GeometricalLineLoad load, RobotSimpleCase sCase, RobotGroupServer rGroupServer)
+        public static void ToRobot(this oM.Structure.Loads.GeometricalLineLoad load, RobotSimpleCase sCase, RobotGroupServer rGroupServer)
```

Comment:
Would add a fall back method accepting `ILoad` at the end of this method, raising a warning that that load type is not supported yet.